runtime/litmus/src/xcm_config.rs

```diff
@@ -149,8 +150,10 @@ impl pallet_xcm::Config for Runtime {
 	// ^ Disable dispatchable execute on the XCM pallet.
 	// Needs to be `Everything` for local testing.
 	type XcmExecutor = XcmExecutor<XcmConfig>;
-	type XcmTeleportFilter = Everything;
-	type XcmReserveTransferFilter = Nothing;
+	type XcmTeleportFilter = Nothing;
+	// This filter here defines what is allowed for XcmExecutor to handle with TransferReserveAsset
+	// Rule.
+	type XcmReserveTransferFilter = Everything;
```

Comment:
Just a question here, so the code change is to enable the reserve transfer and disable teleport, right?

Reply:
Yes. Teleport  means burn/mint for XCM token bridge. Reserve means deposit/withdrawal.